BismarkUnchained/scenes/KinematicBody2D.gd

```diff
@@ -1,5 +1,5 @@
 extends KinematicBody2D
 
 func _physics_process(delta):
-	move_and_collide(Vector2(50, 0))
+	move_and_collide(Vector2(5, 0))
```

Comment:
@shraymittal's movement code will conflict with this, should probably remove the change.